dlt/destinations/athena/athena.py

```diff
@@ -41,7 +41,8 @@
     "timestamp": "timestamp",
     "bigint": "bigint",
     "binary": "binary",
-    "decimal": "decimal(%i,%i)"
+    "decimal": "decimal(%i,%i)",
+    "time": "time"
```

Comment:
looks like athena does not support TIME for parquet based tables (only ICEBERG)
https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg-supported-data-types.html

let's try to use timestamp or string and see if it is able to load. if it cannot load parquet time to str/timestamp then we'll need another destination capability to generate proper parquet files...

Reply:
Neither timestamp/string seems to work as-is:

`time: timestamp`:
```
pyathena.error.OperationalError: SERIALIZATION_ERROR: Could not serialize column 'col11' of type 'timestamp(3)' at position 1:11
```

`time: string`:
```
dlt.destinations.exceptions.DatabaseTerminalException: HIVE_CURSOR_ERROR: Failed to read Parquet file: s3://...
```
So by destination capability you mean some callback or mapping to convert types? There's nothing like this currently, right? Probably would need to convert the time value -> timestamp with dummy date also.

Reply:
let's just block it we do it in Redshift. so if there's a time column in the schema of the table to copy we reject the whole file. 
Please provide a nice explanation in exception and mention that in the docs.

I do not thing we need another mechanism to handle that. Maybe later....